boards/nordic/nrf7002dk/Kconfig.defconfig

```diff
@@ -4,15 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 if BOARD_NRF7002DK_NRF5340_CPUAPP || \
-	BOARD_NRF7002DK_NRF5340_CPUAPP_NS || \
```

Comment:
Everything in this file is related to TF-M and does not make much sense if non-secure targets are not present. It'd be better to remove this file completely.